airlock/business_logic.py

```diff
@@ -485,7 +482,7 @@ def check_status(
                     f"only an output checker can set status to {to_status.name}"
                 )
 
-            if user.username == release_request.author:
+            if user == release_request.author:
```

Comment:
Do we need to define an `__eq__` method for `User` so that they compare by ID or do Django models naturally do the right thing here?